core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -188,6 +221,42 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
       });
   }
 
+  onGroupDelete(rows: Set<WorkflowExecutionsEntry>) {
+    const modalRef = this.modalService.open(DeletePromptComponent);
+    modalRef.componentInstance.deletionType = "execution";
+    let deletionNames = "";
```

Comment:
what is the purpose for this?

Reply:
This is for the deletion confirmation, and I need to extract all of the name for each execution

Reply:
please show one confirmation window only and display count for example are you sure you want to delete 50 executions or are you sure you want to delete all the selected executions

Reply:
To make the execution more userfriendly, if the delete count is less than 10, then I still use the for loop to show the name, if its more than 10, then I show the number count

Reply:
No this becomes repetitive if things are repeated more than 3 times then the human brain finds it annoying. Please show one confirmation

Reply:
I think you misunderstand here, each group deletion will only have confirmation once, so what users will have is one confirmation window with the executions' name listed below

Reply:
Please include only the count and remove unnecessary extra code for not so important feature.